docs/source/development/doc-overview.rst

```diff
@@ -0,0 +1,70 @@
+Contributing to Documentation
+=============================
+
+Using Sphinx
+------------
+IMAP uses `Sphinx <https://www.sphinx-doc.org/en/master/index.html>`_ to generate and maintain documentation. Sphinx allows for automatic usage of Python docstrings, as well features for hierarchical structure, automatic code highlighting, and different output formats.
+
+To add a new documentation file, create a new ``.rst`` file under :file:`docs/source`. RST stands for Restructured Text, which is a type of rich text format used by Sphinx. It is a flexible and powerful format which allows for lots of features which are useful in documentation.
+
+For more information on restructured text, Sphinx has `extensive documentation <https://www.sphinx-doc.org/en/master/usage/index.html>`_.
+
+You can also use Markdown for documentation. Restructured text is the preferred format for documentation, but if you're more comfortable with markdown, that is also acceptable. Markdown documents will use the `MyST parser<https://myst-parser.readthedocs.io/en/latest/index.html>`_.
```

Comment:
```suggestion
You can also use Markdown for documentation. Restructured text is the preferred format for documentation, but if you're more comfortable with markdown, that is also acceptable. Markdown documents will use the `MyST parser <https://myst-parser.readthedocs.io/en/latest/index.html>`_.
```
My local build complains without an extra space before the link

Reply:
Same thing, I must have forgotten to run it one last time before committing! Fixed now